tests/Optimizer/Configuration/AmpStoryCssOptimizerConfigurationTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace AmpProject\Optimizer\Configuration;
+
+use AmpProject\Optimizer\Configuration\AmpStoryCssOptimizerConfiguration;
+use AmpProject\Optimizer\Exception\InvalidConfigurationKey;
+use AmpProject\Optimizer\Exception\InvalidConfigurationValue;
+use AmpProject\Optimizer\Exception\UnknownConfigurationKey;
+use AmpProject\Optimizer\TransformerConfiguration;
+use AmpProject\Tests\TestCase;
+
+/**
+ * Test the AmpStoryCssOptimizerConfiguration.
+ *
+ * @covers \AmpProject\Optimizer\Configuration\AmpStoryCssOptimizerConfiguration
+ * @covers \AmpProject\Optimizer\Configuration\BaseTransformerConfiguration
+ * @package ampproject/amp-toolbox
+ */
+final class AmpStoryCssOptimizerConfigurationTest extends TestCase
+{
+    public function testDefaults()
+    {
+        $configuration = new AmpStoryCssOptimizerConfiguration([]);
+        $this->assertInstanceOf(TransformerConfiguration::class, $configuration);
+        $this->assertEquals(false, $configuration->get('optimizeAmpStory'));
+        $this->assertEquals(false, $configuration->optimizeAmpStory);
```

Comment:
Always use the most specific assertion possible.
```suggestion
        $this->assertFalse($configuration->get('optimizeAmpStory'));
        $this->assertFalse($configuration->optimizeAmpStory);
```